Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -192,7 +192,7 @@ void ThrottleCallback(u64 last_time, s64 cyclesLate)
     const s64 max_fallback = config.iTimingVariance * 1000;
     if (abs(diff) > max_fallback)
     {
-      DEBUG_LOG(COMMON, "system too %s, %ld ms skipped", diff < 0 ? "slow" : "fast",
+      DEBUG_LOG(COMMON, "system too %s, %lld ms skipped", diff < 0 ? "slow" : "fast",
```

Comment:
Use something like `PRIi64`, otherwise you'll get a warning on a different OS instead.